docs/sources/notify/slack/index.md

```diff
@@ -40,6 +40,10 @@ For Open Source Grafana OnCall Slack installation guidance, refer to
 1. Provide your Slack workspace URL and sign with your Slack credentials.
 1. Click **Allow** to give Grafana OnCall permission to access your Slack workspace.
 
+### Configure timeout for acknowledged alerts
+
+You can configure reminders for acknowledged alerts in the Additional settings section of the Chat Ops tab. This section allows you to enable reminders to be sent through Slack at set intervals. You can also choose to have an alert group revert to unaknowledged after a designated period of time if no confirmation is received. 
+
```

Comment:
I think it may not be clear "confirmation" on what is expected? Maybe add something about the reminder being the one that should be confirmed? Other than that, makes sense to me :+1:

Reply:
I changed it to "You can also choose to have an alert group revert to unaknowledged after a designated period of time _if the reminder is not confirmed._"
Is that right?

Reply:
LGTM! Note: unaknowledged -> unacknowledged